### ISSUE TITLE
Visualisation

### ISSUE THREAD

**Issue by octopusCat88** [nagam11/PP2#9]
How to visualise the training and predictions. This will be included in the presentation.

## Training (Validation set)
+ accuracy
+ loss

## Prediction Performance (Test set)
+ accuracy, F1, Q2, ...
+ ROC + AUC
+ AUPR + AUPR
+ Confusion matrix

all as barplots with standard errors
